### ISSUE TITLE
bug(mat-chip): Cannot shift+tab out of a focused mat-chip-listbox

### ISSUE THREAD

**Issue by menuka-s** [angular/components#26698]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

14.2.7

### Description

When focused in a `<mat-chip-listbox>` a user cannot shift+tab to the previous element. 

### Reproduction

Steps to reproduce:
1. Tab to `<mat-chip-listbox>`
2. Try shift+tab to tab out of the `<mat-chip-listbox>`

[Stackblitz](https://stackblitz.com/edit/angular-mdoiwo?file=src%2Fapp%2Fchips-overview-example.html)

### Expected Behavior

A user should be able to press shift+tab to focus back to the previous element.

### Actual Behavior

When pressing shift+tab, the focus remains on a the `<mat-chip-listbox>`. If a value is selected, then the focus returns to that selected `<mat-chip>`.

### Environment

- Angular: 15
- CDK/Material: 15.2
- Browser(s): Chrome, Edge, Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows
